micro_ros_diagnostic_updater/CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ else()
   endif()
 endif()
 
+# build options
+option(MICRO_ROS_DIAGNOSTIC_UPDATER_EXAMPLES "Build example updaters for the micro-ROS diagnostic updater package." TRUE)
```

Comment:
Wouldn't it be better to have them disabled by default? By default, the updater is meant to be on the microcontroller, where we don't want the examples

Reply:
agreed